routes/likeTracker.js

```diff
@@ -59,7 +67,12 @@ router.get('/likes', ensureAuthenticated, (req, res) => {
   LikeTracker.findOne({ userId: req.user.id }, async (err, doc) => {
     try {
       if (err) throw err;
-      if (doc) res.send({ likes: doc['likes'] });
+      if (doc) {
```

Comment:
this is the sort function.  It does not change the array in our DB but sorts the data that is returned by date